app/controllers/api/tasks_controller.rb

```diff
@@ -3,57 +3,36 @@ class Api::TasksController < ApiController
   before_action :set_task, only: [:done, :update, :destroy]
 
   # GET
-  # api/tasks
   def index
     @tasks = Task.on_user
-    # @user = User.where(name: current_user)
-    # @user = :current_user
   end
 
   # POST api/tasks
-  # 期待されるparams = {
-  #   tasks: {
-  #     user_id: int, title: string,
-  #     id_done: false, weight: int,
-  #     deadline_at: "yyyy-mm-dd HH:MM:SS"
-  #   }
-  # }
   def create
     @task = Task.new(task_params)
     if @task.save
-      render json: @task
+      render :task
```

Comment:
__[rubocop]__
* Style/IdenticalConditionalBranches: Move `render :task` out of the conditional.